content/webapp/services/prismic/transformers/json-ld.ts

```diff
@@ -12,8 +12,25 @@ import { Page } from '../../../types/pages';
 import { Season } from '../../../types/seasons';
 import { objToJsonLd } from '@weco/common/utils/json-ld';
 import { Exhibition } from '../../../types/exhibitions';
+import { ExhibitionGuide } from '../../../types/exhibition-guides';
 import { linkResolver } from '@weco/common/services/prismic/link-resolver';
 
+export function exhibitionGuideLd(exhibitionGuide: ExhibitionGuide): JsonLdObj {
+  return objToJsonLd(
+    {
+      name: exhibitionGuide.title,
+      description: exhibitionGuide.relatedExhibition?.description,
+      location: {
+        '@type': 'Place',
+        name: 'Wellcome Collection',
+        address: objToJsonLd(wellcomeCollectionAddress, 'PostalAddress', false),
+      },
+      url: `https://wellcomecollection.org/guides/exhibition/${exhibitionGuide.id}`,
+    },
+    'ExhibtionGuide'
```

Comment:
This isn't a valid type recognised by [schema](https://schema.org/). Not sure what the most appropriate type is though. There's [Guide](https://schema.org/Guide), but the definition doesn't seem right.